client-src/elements/chromedash-gate-chip.js

```diff
@@ -93,6 +99,23 @@ class ChromedashGateChip extends LitElement {
     `];
   }
 
+  _fireEvent(eventName, detail) {
```

Comment:
nit: Could move this to a utility class in a follow-up. This is defined in chromedash-feature-row.js as well

Reply:
Will do.